test/integrations/debugsymbolicator.test.ts

```diff
@@ -334,5 +358,152 @@ describe('Debug Symbolicator Integration', () => {
         },
       });
     });
+
+    it('should symbolicate multiple error with cause ', async () => {
```

Comment:
Let's extend the tests with one more cause in the chain.

Reply:
I Added more tests to check for multiple causes and also if there is a different amount of processed errors compared to the items  found on the hint.